src/bh_route_validators.erl

```diff
@@ -178,5 +179,10 @@ validator_to_json(
         <<"status">> => Status,
```

Comment:
looks like you lose this status field with the later status object using the same key